### PR TITLE
Update whitelist.json

### PR DESCRIPTION
**Pull request by alazymeme**
https://github.com/MrEliasen/twitch-bot-list/issues/9

### CODE REVIEW

File: whitelist.json [hunk 1]
```diff
@@ -4,6 +4,12 @@
     "nightbot",
     "moobot",
     "wizebot",
+    "streamjar",
+    "streamdeckerbot",
+    "vivbot",
+    "dinu",
```
Comment (RAnders00) [inline whitelist.json]: dinu is already in the list

**Comment by MrEliasen** [comment]
Does all these bots have functionality or would any of them be "ok" to ban, like we talked about in #9 with eg. overrustlelogs?

**Comment by alazymeme** [comment]
Since I can't remember which ones I changed, please see the below list:

streamjar - chatbot http://streamjar.tv/
streamdeckerbot - only added through https://www.streamdecker.com/
vivbot - chatbot http://vivbot.com/
streamkit - chatbot https://streamkit.gg/app
tipeeebot - afaik bot for tipeeestream https://www.tipeeestream.com/

File: whitelist.json (hunk 1 above)
Comment (alazymeme) [reply whitelist.json]: yeah ik but I removed it then readded it

**Comment by MrEliasen** [comment]
Gotcha! Alright thank for the details.

Could you remove any of the bots you added from the `list.json`, I would be happy to accept the PR after that!

**Comment by alazymeme** [comment]
Done, forgot to add `wzbot` to the list

**Comment by MrEliasen** [comment]
Done and done, thank you bud!

**Pull request merged**